forms.rst

```diff
@@ -560,46 +560,51 @@ To see the second approach - adding constraints to the form - and to
 learn more about the validation constraints, please refer to the
 :doc:`Symfony validation documentation </validation>`.
 
+Friendlier validation messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. versionadded:: 5.2
 
-    In Symfony 5.2, the form validation messages have been rewritten to be more
-    user-friendly. Set the ``legacy_error_messages`` option to ``false`` to
-    enable these new messages:
+    The ``framework.form.legacy_error_messages`` option was added in Symfony 5.2
```

Comment:
We always try to use "[...] was introduced in Symfony 5.2." in all our versionadded directives.